code/modules/ruins/spaceruin_code/forgottenship.dm

```diff
@@ -0,0 +1,115 @@
+// forgottenship ruin
+GLOBAL_VAR_INIT(fscpassword, generate_password())
+
+/proc/generate_password()
+	return "[pick(GLOB.phonetic_alphabet)] [rand(1,1000)]"
```

Comment:
By the way, are you sure that saying "Alpha 1000" won't open the door if the password is "Alpha 100"? I genuinely don

Honestly, that's such an edge case that I wouldn't bother addressing it (and addressing it would probably make it impossible for the door to open for a message if the message is embedded in another sentence (for example: "Door, open. Authorization code: Alpha 100.").

Reply:
I was testing it, saying only the code itself works.
You can also just input it by using the blast-door.

Reply:
I guess it requires to say the specific code, 100 won't work if it's 1000.